services/distribution/src/main/java/app/coronawarn/server/services/distribution/statistics/keyfigurecard/factory/HeaderCardFactory.java

```diff
@@ -23,6 +24,11 @@
   public KeyFigureCard makeKeyFigureCard(StatisticsJsonStringObject stats) {
     DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
     LocalDate dateTime = LocalDate.parse(stats.getEffectiveDate(), formatter);
+    if (this.getCardId() == HOSPITALIZATION_INCIDENCE_CARD.ordinal()) {
+      dateTime = stats.getHospitalizationEffectiveDate() == null || stats.getHospitalizationEffectiveDate().isEmpty()
+          ? LocalDate.parse(stats.getEffectiveDate(), formatter) :
+          LocalDate.parse(stats.getHospitalizationEffectiveDate(), formatter);
+    }
```

Comment:
@ioangut dateTime is already initialized with `stats.getEffectiveDate`, so it's sufficient to overwrite only in case it's the correct card and the value isn't empty... what do you think?

```suggestion
    if (this.getCardId() == HOSPITALIZATION_INCIDENCE_CARD.ordinal() && !ObjectUtils.isEmpty(stats.getHospitalizationEffectiveDate())) {
      dateTime = LocalDate.parse(stats.getHospitalizationEffectiveDate(), formatter);
    }
```

Reply:
I agree , Thank you for suggestion!